### PR TITLE
Use Kjar to get GAV for container to be deployed to the Workbench

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/Kjar.java [hunk 1]
```diff
@@ -48,6 +50,10 @@ public String getVersion() {
         return version;
     }
 
+    public ReleaseId getKieServerReleaseId() {
```
Comment (jhrcek) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/Kjar.java]: Nice improvement. I'm just a bit confused by the name. KieServerReleaseId sounds to me like the ReleaseId of KieServer. Do we need "KieServer" in the name? If we name it just `getReleaseId` can we confuse it with something else?

Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/Kjar.java]: Problem is that there are 2 different classes called ReleaseId. One is `org.kie.api.builder.ReleaseId`, used in engine, the other is `org.kie.server.api.model.ReleaseId`. I wanted to distinguish it somehow.

Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/Kjar.java]: After some thoughts I decided to remove this method, it brings unnecessary dependency on Kie server API to this class.

**Comment by sutaakar** [comment]
Removed getKieServerReleaseId() method to keep the Kjar enum without unneeded dependencies.

**Comment by sutaakar** [comment]
Test failures not related

**Pull request merged**